src/sockjs.coffee

```diff
@@ -123,7 +124,8 @@ class Listener
 
     handler: (req, res, extra) =>
         # All urls that match the prefix must be handled by us.
-        if not req.url.match(@path_regexp)
+        url = urlParser.parse(req.url).pathname
```

Comment:
With the import change, let's modify these lines to `urlPath = url.parse(req.url).pathname` and `if not urlPath.match(@path_regexp)`.